src/main/java/Mario/Mario.java

```diff
@@ -0,0 +1,232 @@
+        String logo = ui.getLine()
+                + "It's-a me, Mario! \n"
+                + "How can I help you today?\n"
+                + ui.getLine();
+
+        System.out.println(logo);
+
+        Scanner scanner = new Scanner(System.in);
+        String input;
+
+        do {
+            input = scanner.nextLine();
+            parser.parseCommand((input));
+
+        }
+        while (!input.equals("bye") && !input.equals("Bye"));
+
+        scanner.close();
+
+    }
+
+    /**
+     * Returns a formatted list of tasks.
+     */
+    public static void listTasks() {
+        int len = lst.length();
+        if (len == 0) {
+            System.out.println(ui.betweenLines(
+                    "There's nothing panned yet!"));
+        } else {
+            System.out.println(ui.getLine()
+                    + "     Here you go:");
+
+            for (int i = 0; i < len; i++) {
+                int order = i + 1;
+                System.out.println("     " + order + ". "
+                        + lst.getTask(order).getName());
+            }
+            System.out.println(ui.getLine());
+        }
+    }
+
+    /**
+     * Adds a todo task to the Task list
+     * @param name of task
+     * @throws IOException
+     */
+    public static void addTodoTask(String name) throws IOException {
+        ToDo todo = new ToDo(name);
+        lst.addTask(todo);
+        saveToStorage(lst);
+        System.out.println(ui.betweenLines(
+                 "All right! I've added this task: \n"
+                + "        " + todo.getName() + "\n"
+                + "     Now you have " + lst.length()
+                + " task(s) in your list!\n"));
+    }
+
+
+    /**
+     * Adds a deadline task to the task list.
+     * @param name
+     * @param deadline
+     * @throws IOException
+     */
+    public static void addDeadlineTask(String name, String deadline) throws IOException {
+        Deadline dl = new Deadline(name, deadline);
+        lst.addTask(dl);
+        saveToStorage(lst);
+        System.out.println(ui.betweenLines(
+                "All right! I've added this task: \n"
+                + "        " + dl.getName() + "\n"
+                + "     Now you have " + lst.length()
+                        + " task(s) in your list!\n"));
+    }
+
+    /**
+     * Adds an event task to the task list.
+     * @param name
+     * @param start Starting time of the event.
+     * @param end Ending time of the event.
+     * @throws IOException
+     */
+    public static void addEventTask(String name, String start, String end) throws IOException {
+        Event event = new Event(name, start, end);
+        lst.addTask(event);
+        saveToStorage(lst);
+        System.out.print(ui.betweenLines(
+                "All right! I've added this task: \n"
+                + "        " + event.getName() + "\n"
+                + "     Now you have " + lst.length() + " task(s) in your list!\n"));
+    }
+
+    public int getNumTasks() {
```

Comment:
Perhaps naming the function getSize or size would make its purpose clearer to future reviewers